File: strings/testCpy/testCpy.js
const { argv } = require('process')
const readline = require('readline')

const strings = []
const n = argv[2]
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
  terminal: false
});


rl.on('line', (line) => {
    strings.push(line)
});

rl.once('close', () => {
    const t1 = process.hrtime()[1]
    let buffer = '';
    for(let i = 0; i < n; i++){
        buffer = strings[i]
    }
    const t2 = process.hrtime()[1]
    console.log((t2 - t1) / 1000000)
});

File: strings/testCpy/testCpy.py
from time import time
from sys import argv

def strcat():
    n = int(argv[1])
    strings = []
    for i in range(0, n):
        strings.append(input(''))

    t1 = time()
    buffer = ''
    for str in strings:
        buffer = str
    t2 = time()
    print(t2 - t1)


if( __name__ == '__main__'):
    strcat()